pom.xml

```diff
@@ -92,6 +92,18 @@
       <artifactId>throttle-concurrents</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.10.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
The plugin is already using JUnit 4.  Please stay with JUnit 4.  If JUnit 5 is required, please use the most recent JUnit 5 release instead of an older release.
```suggestion
```

Reply:
Changed it back to use JUnit 4.

Reply:
Thanks!